addons/l10n_it/migrations/0.7/post-map_carryover_to_new_report.py

```diff
@@ -0,0 +1,82 @@
+from odoo import api, SUPERUSER_ID, tools
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    vat_report_id = env.ref('l10n_it.tax_report_vat').id
+    monthly_vat_report_id = env.ref('l10n_it.tax_monthly_report_vat').id
+
+    external_value_cols = [
+        col
+        for col in tools.table_columns(env.cr, 'account_report_external_value')
+        if col not in ['id', 'carryover_origin_report_line_id']  # TODO what about target_report_expression_id?
```

Comment:
I guess I simply wasn't sure of everything since I was only writing it in the text editor of GitHub without any testing or references to the IDE :wink:

Reply:
Well then i'm not sure there is anything special to be done with that column. At least, right now, I can't see in which way an exception would be required.